### PR TITLE
Update the api description of the Ray class

### PR DESCRIPTION
**Pull request by GengineJS**

### CODE REVIEW

File: cocos/core/geometry/ray.ts [hunk 1]
```diff
@@ -42,13 +40,13 @@ export class Ray {
...
      * @zh
      * 创建一条射线。
-     * @param {number} ox 起点的 x 部分。
-     * @param {number} oy 起点的 y 部分。
-     * @param {number} oz 起点的 z 部分。
-     * @param {number} dx 方向的 x 部分。
-     * @param {number} dy 方向的 y 部分。
-     * @param {number} dz 方向的 z 部分。
-     * @return {Ray} 射线。
+     * @param {number} ox @en Start position x @zh 起点的 x 部分。
```
Comment (minggo) [inline cocos/core/geometry/ray.ts]: x component of start point?

File: cocos/core/geometry/ray.ts [hunk 2]
```diff
@@ -59,8 +57,8 @@ export class Ray {
      * Creates a new ray initialized with values from an existing ray
      * @zh
      * 从一条射线克隆出一条新的射线。
-     * @param {Ray} a 克隆的目标。
-     * @return {Ray} 克隆出的新对象。
+     * @param {Ray} a @en Cloned target @zh 克隆的目标。
```
Comment (minggo) [inline cocos/core/geometry/ray.ts]: The Ray object to be cloned from?

File: cocos/core/geometry/ray.ts [hunk 3]
```diff
@@ -90,10 +88,10 @@ export class Ray {
      * create a ray from two points
      * @zh
      * 用两个点创建一条射线。
-     * @param {Ray} out 接受操作的射线。
-     * @param {Vec3} origin 射线的起点。
-     * @param {Vec3} target 射线上的一点。
-     * @return {Ray} out 接受操作的射线。
+     * @param {Ray} out @en Ray that accepts operations @zh 接受操作的射线。
```
Comment (minggo) [inline cocos/core/geometry/ray.ts]: The Ray object to be modified.

**Review by star-e — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**